config/uat/bridgeserver2.yaml

```diff
@@ -26,6 +26,7 @@ parameters:
   HibernateConnectionUsessl: 'true'
   NewRelicAppName: bridgeserver2-uat
   NewRelicLicenseKey: !ssm /infra/NewRelicLicenseKey
+  ParticipantFileBucket: org-sagebridge-participantfile-uat
```

Comment:
Should I remove these? Aren't the bucket names now basically randomly generated by CloudFormation?

Reply:
yes, you don't need to pass in the name you are already referring to it.